internal/exec/util/file.go

```diff
@@ -184,6 +184,10 @@ func (u Util) SetPermissions(mode *int, node types.Node) error {
 func (u Util) PerformFetch(f FetchOp) error {
 	path := f.Node.Path
 
+	if _, err := os.Stat(path); !os.IsNotExist(err) {
+		panic(fmt.Sprintf("path %s already exists; can't overwrite", path))
```

Comment:
Since this is an unusual behavior, I thought of using `panic` here.  What other folks think about it?